src/utils/properties.js

```diff
@@ -64,24 +64,19 @@ export const renderProperty = (prop, index) => {
 	}
 };
 
-const renderHocConfig = (config) => {
+const renderHocConfig = async (config) => {
 	return (
 		<section className={css.hocconfig}>
 			<h5>Configuration</h5>
 			<dl>
-				{config.members.static.map(renderProperty)}
+				{await Promise.all(config.members.static.map(renderProperty))}
 			</dl>
 		</section>
 	);
 };
 
 const propSort = (a, b) => {
-	let aIsRequired = hasRequiredTag(a);
-	let bIsRequired = hasRequiredTag(b);
-
-	if (aIsRequired !== bIsRequired) {
-		return aIsRequired ? -1 : 1;
-	} else if (a.name < b.name) {
+	if (a.name < b.name) {
```

Comment:
Note: `propSort` method had to be refactor since it is calling `hasRequiredTag` async function. sort array method does not work with promises, I left only the sorting algorithm,
and extracted the required tags of properties on `renderInstanceProperties` - line 109